.buildkite/test-pipeline.yaml

```diff
@@ -632,6 +632,7 @@ steps:
   # we can only upgrade after this is resolved
   # TODO(jerryzh168): resolve the above comment
   - uv pip install --system torchao==0.13.0 --index-url https://download.pytorch.org/whl/cu129
+  - uv pip install --system conch-triton-kernels
```

Comment:
Why is this needed now if we didn't need this before? Is it needed for the new model somehow?